templates/datatransport/Project.cshtml

```diff
@@ -0,0 +1,116 @@
+            <PackageTags>xamarin, android, bindings, google, datatransport, transport, api </PackageTags>
+            break; 
+          case "Xamarin.Google.Android.DataTransport.TransportBackendCct":
+            <Title>Xamarin.Google.Android.DataTransport.TransportBackendCct</Title>
+            <PackageDescription>Bindings for Xamarin Google.Android.DataTransport.TransportBackendCct package</PackageDescription>
+            <Summary>Bindings for Xamarin Google.Android.DataTransport.TransportBackendCct package</Summary>
+            <RequireLicenceAcceptance>true</RequireLicenceAcceptance>
+            <PackageRequireLicenseAcceptance>true</PackageRequireLicenseAcceptance>
+            <PackageTags>xamarin, android, bindings, google, datatransport, transport, backend, cct </PackageTags>
+            break; 
+          case "Xamarin.Google.Android.DataTransport.TransportRuntime":
+            <Title>Xamarin.Google.Android.DataTransport.TransportRuntime</Title>
+            <PackageDescription>Bindings for Xamarin Google.Android.DataTransport.TransportRuntime package</PackageDescription>
+            <Summary>Bindings for Xamarin Google.Android.DataTransport.TransportRuntime package</Summary>
+            <RequireLicenceAcceptance>true</RequireLicenceAcceptance>
+            <PackageRequireLicenseAcceptance>true</PackageRequireLicenseAcceptance>
+            <PackageTags>xamarin, android, bindings, google, datatransport, transport, runtime </PackageTags>
+            break; 
+          default:
+            break; 
+      }
+      <PackageVersion>@(Model.NuGetVersion)</PackageVersion>
+    }
+  </PropertyGroup>
+
+  <ItemGroup>
+    <TransformFile Include="..\..\source\@(Model.MavenArtifacts[0].MavenGroupId)\@(Model.MavenArtifacts[0].MavenArtifactId)\Transforms\*.xml" Link="Transforms\%(Filename)%(Extension)" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <Compile Include="..\..\source\@(Model.MavenArtifacts[0].MavenGroupId)\@(Model.MavenArtifacts[0].MavenArtifactId)\Additions\*.cs" Link="Additions\%(Filename)%(Extension)" />
+  </ItemGroup>
+
+  <!--
+  <ItemGroup>
+    @foreach (var art in @Model.MavenArtifacts) 
+    {
+    <LibraryProjectZip Include="..\..\externals\@(art.MavenGroupId)\@(art.MavenArtifactId).aar" />
+    <JavaDocJar Include="..\..\externals\@(art.MavenGroupId)\@(art.MavenArtifactId)-javadoc.jar"/>
+    }
+  </ItemGroup>
+  -->
+  
+  <ItemGroup>
+    @foreach (var dep in @Model.NuGetDependencies) 
+    {
+      if (dep.IsProjectReference) 
+      {
+    <ProjectReference Include="..\..\generated\@(dep.MavenArtifact.MavenGroupId).@(dep.MavenArtifact.MavenArtifactId)\@(dep.MavenArtifact.MavenGroupId).@(dep.MavenArtifact.MavenArtifactId).csproj" PrivateAssets="none" />
+      } 
+      else 
+      {
+        if (dep.NuGetPackageId == "Xamarin.AndroidX.Annotation")
+        {
+          <PackageReference Include="@(dep.NuGetPackageId)" Version="1.2.0.3" />
```

Comment:
What is this for?  Does the automatic dependency resolution have a bug?

Reply:
transitive dependencies cause "Detected package downgrade" warnings and errors. Explicitly adding nuget with version is workaround for that

Reply:
didn't want to use something like:

```
 <PropertyGroup>
    <NoWarn>$(NoWarn);NU1605</NoWarn>
  </PropertyGroup>
```